src/services/statistic/index.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const service = require('feathers-mongoose');
+const hooks = require('./hooks/index');
+const schoolModel = require('../school/model');
+const userModel = require('../user/model');
+const accountModel = require('../account/model');
+const homeworkModel = require('../homework/model');
+const lessonModel = require('../lesson/model');
+const groupModel = require('../user-group/model');
+const fileModel = require('../fileStorage/model');
+
+class StatisticsService {
+	find({query, payload}) {
+		const userPromise = userModel.count().exec();
+		const schoolPromise = schoolModel.count().exec();
+		const accountPromise = accountModel.count().exec();
+		const homeworkPromise = homeworkModel.homeworkModel.count().exec();
+		const submissionPromise = homeworkModel.submissionModel.count().exec();
+		const commentPromise = homeworkModel.commentModel.count().exec();
+		const lessonPromise = lessonModel.count().exec();
+		const classPromise = groupModel.classModel.count().exec();
+		const coursePromise = groupModel.courseModel.count().exec();
+		const teacherPromise = userModel.count({roles: '0000d186816abba584714c98'}).exec();
+		const studentPromise = userModel.count({roles: '0000d186816abba584714c99'}).exec();
+		const filePromise = fileModel.fileModel.count().exec();
+		const directoryPromise = fileModel.directoryModel.count().exec();
+
+		return Promise.all([userPromise, schoolPromise, accountPromise, homeworkPromise, submissionPromise, commentPromise, lessonPromise, classPromise, coursePromise, teacherPromise, studentPromise, filePromise, directoryPromise])
+			.then(res => {
+				res = res.map((result, i) => {
```

Comment:
Kann man vielleicht aus dem model irgendwie den namen der collection aus dem Model ziehen anstatt ein riesiges switch-case zu machen? https://stackoverflow.com/questions/43331706/how-to-get-collection-name-from-a-mongoose-model-object

Dann vielleicht nen array aus objekten der form 
`
{
name: 'users',
promise: userPromise
}
`
 
machen und dann über dieses array, was z.B. `promises` heißt iterieren und sowas wie:

`
return Promise.all(promises.map(p => {
 return p.promise.then(response => {
   let result = {};
  result[p.name] = response;
  return result;
}
});
`

Dann spart man sich irgendwie diesen riesigen switch-case. Kannst ja mal probieren, ob das irgendwie so geht.